src/utils/getParents.tsx

```diff
@@ -0,0 +1,17 @@
+import { Person } from '../types';
+
+export function getParents(people: Person[]) {
+  const peopleWithParents = people.map(person => {
+    const newPerson = { ...person };
+
+    newPerson.father = people
+      .find(father => father.name === newPerson.fatherName);
+
+    newPerson.mother = people
+      .find(mother => mother.name === newPerson.motherName);
+
+    return newPerson;
+  });
+
+  return peopleWithParents;
```

Comment:
why do you need this function?

Reply:
tried to use links to parents and forgot to delete